pkgs/by-name/ya/yazi/flavors/gruvbox-dark/default.nix

```
@@ -0,0 +1,24 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+stdenv.mkDerivation rec {
+  name = "gruvbox-dark";
+  src = fetchFromGitHub {
+    owner = "bennyyip";
+    repo = "${name}.yazi";
+    rev = "91fdfa70f6d593934e62aba1e449f4ec3d3ccc90";
+    hash = "sha256-RWqyAdETD/EkDVGcnBPiMcw1mSd78Aayky9yoxSsry4=";
+  };
```

- It's not recommended to use name/pname in source since changing the former will also change the latter, which isn't what we want
- A newer commit is available

```suggestion
  src = fetchFromGitHub {
    owner = "bennyyip";
    repo = "gruvbox-dark.yazi";
    rev = "2c61c90e8725136760e5d12b7ebc4c24d602241b";
    hash = "sha256-qNxauqv8tRPztO17PGKl3ue6cSWGqIuEzYViaRXDVcQ=";
  };
```